feat: add support for the namespace file resource and datasource (#95)
kestra-io/terraform-provider-kestra: docs/data-sources/namespace_file.md

---
# generated by https://github.com/hashicorp/terraform-plugin-docs
page_title: "kestra_namespace_file Data Source - terraform-provider-kestra"
subcategory: ""
description: |-

>   Use this data source to access information about an existing Namespace File

Reviewer: ```suggestion
  Use this data source to manage Namespace File resource
```

---

# kestra_namespace_file (Data Source)

> Use this data source to access information about an existing Namespace File

Reviewer: ```suggestion
Use this data source to manage the Namespace File resource
```

## Example Usage

```terraform
data "kestra_namespace_file" "example" {
  namespace_ = "io.kestra.mynamespace"
  filename   = "my-file.yml"
}
```

<!-- schema generated by tfplugindocs -->
## Schema

### Required

- `filename` (String) The path to the namespace file that will be created.
Missing parent directories will be created.
If the file already exists, it will be overridden with the given content.

> - `namespace` (String) The Namespace file namespace.

Reviewer: ```suggestion
- `namespace` (String) The namespace to which the namespace file resource should be deployed
```
